Modules/mathmodule.c

```diff
@@ -1032,7 +1040,13 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 #define FUNC1(funcname, func, can_overflow, docstring)                  \
     static PyObject * math_##funcname(PyObject *self, PyObject *args) { \
-        return math_1(args, func, can_overflow);                            \
+        return math_1(args, func, can_overflow, NULL);                  \
+    }\
```

Comment:
```suggestion
        return math_1(args, func, can_overflow, NULL);                  \
    }                                                                   \
```

@picnixz, you meant this and ditto for the new macro?

Reply:
I doubt it's a good idea, see other macros nearby.

Reply:
Ah yes that's what I meant. I'm sorry I am on a phone so it's hard to see the context and whether this would conform to the overall style. If it does not, forget my comment